src/utils/keychain.js

```diff
@@ -48,38 +48,41 @@ export type KeyChainData = {
   pin?: ?string,
 };
 
-export const handleCatch = (accountAddress: ?string, error: ?any) => {
-  reportErrorLog('Exception caught on keychain: ', error);
-  const colors = getThemeColors();
-  const buttons = [];
-  buttons.push({
-    text: t('error.failedKeychain.exitButtonText'),
-    onPress: () => RNExitApp.exitApp(),
-  });
-  accountAddress = accountAddress ?? etherspotService?.getAccountAddress(CHAIN.ETHEREUM);
-  if (accountAddress) {
+export const handleCatch = (accountAddress: ?string, error: ?any[]) => {
+  const msg = error ? error[0]?.Error?.msg : '';
```

Comment:
Could you just `.toString()` `error` here?

Reply:
that's what I thought but it won't return as json to separate code and then have to parseJson and then take msg